pkg/compare/compare_test.go

```
@@ -338,9 +338,23 @@ func defaultTest(name string) Test {
 }
 
 func matchErrorRegexCheck(msg string) Check {
+	// Trim the optional `(?s)` flag prefix if it's there so we can wrap the whole thing
+	cleanMsg := msg
+	hasDotAll := strings.HasPrefix(msg, "(?s)")
+	if hasDotAll {
+		cleanMsg = strings.TrimPrefix(msg, "(?s)")
+	}
+	
```

golangci-lint (2.13.2)

[error] File is not properly formatted

(gofmt)